### ISSUE TITLE
SolastaBardClass.dll

### ISSUE THREAD

**Issue by Dragon5e** [Holic75/SolastaBard#4]
Request you add this dll to the latest release. I had to add it from bard version SolastaBardClass.zip which is 1.05 version? I'm using this in Unity mod manager. So far I added in the monk and bard dll. Warlock is next.

**Comment by Carden-Gixoth** [comment]
How did you get the bard class, and the monk class, and the warlock, assuming you have, to work? And do they work on Solasta v1.2.15? I'm trying to get them to work, but for the life of me, I can't figure them out.

**Comment by Dragon5e** [comment]
Never did. I got all the packages and then the game refuse to load them at all. Afraid to say those class are stuck in the past.


     On Thursday, February 10, 2022, 05:09:52 PM CST, Austin ***@***.***> wrote:  
 
 


How did you get the bard class, and the monk class, and the warlock, assuming you have, to work? And do they work on Solasta v1.2.15? I'm trying to get them to work, but for the life of me, I can't figure them out.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Carden-Gixoth** [comment]
Yeah, that's the same issue I had. Even tried unzipping, copying over the newest update, replacing all conflicting files, rezipping and calling it good, but that didn't fix it. Game kept saying it was corrupted, uninstall the Bard, and it works properly.
